### ISSUE TITLE
How to get function?

### ISSUE THREAD

**Issue by hanusek** [pleiszenburg/zugbruecke#71]
Hello.
How to bind function?

```python
import zugbruecke
from zugbruecke import *
from zugbruecke.wintypes import *
from zugbruecke.core import *

dll_path = 'my_path_foo_bar.dll'
dll = zugbruecke.windll.LoadLibrary(dll_path)
connect = dll.K5DB_Connect
connect.restype = zugbruecke.c_bool
connect.argtypes = ...
```

```c
DWORD __declspec(dllexport) K5DB_Connect (		
	HWND hwndCallback,					// handle of the parent window for callbacks
	DWORD msgCallback,					// wished callback message
	DWORD dwFlags,						// 1 if self-notifications wished
	DWORD dwData,						// reserved for extension
	LPCSTR szClientName					// client name
	); // RETURN: client ID or NULL if error
```

**Comment by s-m-e** [comment]
First of all, [do not use wildcard imports in Python](https://www.reddit.com/r/learnpython/comments/8zej2c/eli5_what_are_wildcard_imports_and_why_should_i/). There are [legitimate edge cases](https://stackoverflow.com/questions/3615125/should-wildcard-import-be-avoided) where they make sense - but this is not.

Second, there is no need to import anything from `zugbruecke.core`. The core contains intentionally undocumented internal APIs.

Your code should approximately look like this:

```python
import zugbruecke as ctypes
from zugbruecke.wintypes import (
    HWND, # ctypes.c_void_p
    DWORD, # ctypes.c_ulong
    LPCSTR, # ctypes.c_char_p
)

dll_path = 'my_path_foo_bar.dll'
dll = ctypes.cdll.LoadLibrary(dll_path) # very likely `cdll` because of `__declspec`
connect = dll.K5DB_Connect
connect.argtypes = (
    HWND, # handle of the parent window for callbacks
    DWORD, # wished callback message
    DWORD, # 1 if self-notifications wished
    DWORD, # reserved for extension
    LPCSTR, # client name
)
connect.restype = DWORD
```

**Comment by hanusek** [comment]
```
handle = (HWND)(0)
message = (DWORD)(0)
sn = (DWORD)(1)
reserved = (DWORD)(0)
path = (LPCSTR)(b'/home/mhanusek/work/ModbusTestProject')

return_value = connect(handle, message, sn, reserved, path)
```
Error:
```
Traceback (most recent call last):
  File "./K5TEST.py", line 43, in <module>
    return_value = connect(handle, message, sn, reserved, path)
  File "/home/mhanusek/.local/lib/python3.6/site-packages/zugbruecke/core/routine_client.py", line 151, in __call__
    raise return_dict['exception']
ValueError: Procedure probably called with too many arguments (20 bytes in excess)
```

**Comment by hanusek** [comment]
Works. Thanks.
